python/pdstools/plots/plot_base.py

```diff
@@ -815,20 +977,28 @@ def plotPredictorPerformanceHeatmap(
             facets=facets,
             last=True,
         )
-        df = df.filter(pl.col("PredictorName") != "Classifier")
+        df = df.filter(pl.col("PredictorName") != "Classifier").with_column(
+            pl.col("PerformanceBin") * 100
+        )
+        df, by = self._generateFacets(df, by)
 
         # TODO: implement facets.
-        df = self.pivot_df(df, by=by)
+        df = self.pivot_df(df, by=by[0])
         if top_n > 0:
-            df = df[0:top_n]
+            df = df[: top_n + 1, : top_n + 1]
```

Comment:
Slicing the dataframe with .head() instead of [: , :] usage was not possible since the predictor names and we are in columns instead of rows after pivoting them. Maybe we can integrate top_n inside pivot_df to achieve that.

[See commit](https://github.com/yusufuyanik1/pega-datascientist-tools/commit/e64af014c836da72859687f329d4ccc6d27606a6)

Reply:
Repeated from previous - probably can just use a select

Reply:
Done